banners/mobile_english/C24_WMDE_Mobile_EN_00/styles/styles.scss

```diff
@@ -0,0 +1,16 @@
+@use '../../../../src/components/BannerConductor/banner-transition';
+@use '../../../../src/themes/UseOfFunds/UseOfFunds';
+@use '../../../../src/themes/Yperala/defaults';
+@use 'Banner';
+@use 'MiniBanner';
+@use 'FullPageBanner';
+@use '../../../../src/themes/Yperala/Slider/Slider';
+@use '../../../../src/themes/Yperala/ProgressBar/ProgressBar';
+@use '../../../../src/themes/Yperala/DonationForm/MultiStepDonation';
+@use '../../../../src/themes/Yperala/DonationForm/SubComponents/SelectGroup';
+@use '../../../../src/themes/Yperala/DonationForm/SubComponents/SelectGroupImageLabel';
```

Comment:
you created a new path alias for "@banners", 
I wonder if we could also have an alias for the path to the `src` folder, because those are quite long

Reply:
The existing src alias already works, I did a search and replace.